jboss-kie-workbench/configure.sh

```diff
@@ -16,8 +16,12 @@ chown -R jboss:root ${JBOSS_HOME}/bin/
 chmod -R g+rwX ${JBOSS_HOME}/bin/
 
 # Ensure that the local data directory exists
-DATA_DIR=/opt/kie/data
+DATA_DIR=${JBOSS_HOME}/standalone/data
+KIE_HOME_DIR=/opt/kie
```

Comment:
What about?
KIE_DATA_DIR=/opt/kie/data
... etc ...

Reply:
JBOSS_DATA_DIR ?
KIE_DATA_DIR ?

Reply:
This dir is manipulated by the PV. If in the future we create another one like /opt/kie/cert, the permissions will inherit.

Reply:
sounds good.